### PR TITLE
Role updates

### PR DESCRIPTION
**Pull request by bryan-bar**
- `benchmark-steps` role defaults updated
- `benchmark_name` set from `benchmark_directory`
- `provision` tasks added
- `unprovision` tasks added
- Pause for user input during validation step by setting the environment variable `INTERACTIVE=true`
- `.gitignore` updated

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,5 @@ servers.yml
 # Generated files
 environment.yml
 infrastructure.yml
+credentials.yml
+*results*/
```
Comment (mw2q) [inline .gitignore:13]: I'm not sure if we need the wildcards around results.  Are things making variations on a results dir?

File: aws-edb-postgres-tuner-dbt2/input.yml [hunk 2]
```diff
@@ -17,7 +17,7 @@ pg_ssh_user: rocky
 dbt2_duration: 600
 dbt2_warehouse: 3600
 dbt2_connections: 72
-dbt2_rampup: 15
+dbt2_rampup: 45
```
Comment (mw2q) [inline aws-edb-postgres-tuner-dbt2/input.yml:20]: We want this to stay 15.

File: aws-lsr-3-nodes/input.yml [hunk 3]
```diff
@@ -15,7 +15,7 @@ pg_image_owner: 679593333241
 pg_ssh_user: rocky
 tpcc_duration: 20
 tpcc_warehouse: 1000
-tpcc_rampup: 1
+tpcc_rampup: 45
```
Comment (mw2q) [inline aws-lsr-3-nodes/input.yml:18]: I think this was designed for just a 1 min rampup...

File: aws-pgd-3-nodes/input.yml [hunk 4]
```diff
@@ -25,7 +25,7 @@ dbt2_warehouse: 3600
 dbt2_connections_min: 5
 dbt2_connections_max: 100
 dbt2_connections_step: 5
-dbt2_rampup: 1
+dbt2_rampup: 45
```
Comment (mw2q) [inline aws-pgd-3-nodes/input.yml:28]: This one should stay 1.

File: aws-psr-3-nodes/input.yml [hunk 5]
```diff
@@ -15,7 +15,7 @@ pg_image_owner: 679593333241
 pg_ssh_user: rocky
 tpcc_duration: 20
 tpcc_warehouse: 1000
-tpcc_rampup: 1
+tpcc_rampup: 45
```
Comment (mw2q) [inline aws-psr-3-nodes/input.yml:18]: This one should stay 1.

File: aws-tpcc-pgd/input.yml [hunk 6]
```diff
@@ -13,7 +13,7 @@ pg_image_owner: 679593333241
 pg_ssh_user: rocky
 tpcc_duration: 20
 tpcc_warehouse: 2000
-tpcc_rampup: 1
+tpcc_rampup: 45
```
Comment (mw2q) [inline aws-tpcc-pgd/input.yml:16]: This one should stay 1.

File: aws-tpcc-refdata/input.yml [hunk 7]
```diff
@@ -13,7 +13,7 @@ pg_image_owner: 679593333241
 pg_ssh_user: rocky
 tpcc_duration: 20
 tpcc_warehouse: 2000
-tpcc_rampup: 1
+tpcc_rampup: 45
```
Comment (mw2q) [inline aws-tpcc-refdata/input.yml:16]: This one should stay 1.

File: aws-tpcc-tde-medium-amd/input.yml [hunk 8]
```diff
@@ -20,7 +20,7 @@ pgwal_storage_size: 800
 pgwal_storage_iops: 12000
 tpcc_duration: 20
 tpcc_warehouse: 2000
-tpcc_rampup: 1
+tpcc_rampup: 45
```
Comment (mw2q) [inline aws-tpcc-tde-medium-amd/input.yml:23]: This one should stay 1.

File: aws-tpcc-tde/input.yml [hunk 9]
```diff
@@ -13,7 +13,7 @@ pg_image_owner: 679593333241
 pg_ssh_user: rocky
 tpcc_duration: 20
 tpcc_warehouse: 2000
-tpcc_rampup: 1
+tpcc_rampup: 45
```
Comment (mw2q) [inline aws-tpcc-tde/input.yml:16]: This one should stay 1.

File: aws-tpcc-xdb/input.yml [hunk 10]
```diff
@@ -16,7 +16,7 @@ pg_ssh_user: rocky
 xdb_instance_type: c5.4xlarge # Assumes attached storage devices from instance type
 tpcc_duration: 5
 tpcc_warehouse: 2000
-tpcc_rampup: 1
+tpcc_rampup: 45
```
Comment (mw2q) [inline aws-tpcc-xdb/input.yml:19]: This one should stay 1.

**Review by mw2q — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (bryan-bar) [reply .gitignore]: No variations are being made.
Changed to `results/`

File: aws-edb-postgres-tuner-dbt2/input.yml (hunk 2 above)
Comment (bryan-bar) [reply aws-edb-postgres-tuner-dbt2/input.yml]: rampup commit removed

**Review by mw2q — APPROVED** [review]
this looks sane to me

**Pull request merged**